spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/controller/TaskSchedulerControllerTests.java

```diff
@@ -17,11 +17,13 @@
 package org.springframework.cloud.dataflow.server.controller;
 
 import java.net.URI;
+import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 
 import org.junit.Before;
+import org.junit.Ignore;
```

Comment:
Can we remove the Ignore on these tests now or is there anything to be addressed?